### DISCUSSION TITLE
How to manage commands thar require Confirmations in Scrapli?

### DISCUSSION THREAD

**Discussion by codingnetworksb** [scrapli/scrapli_community#182] [Q&A]
Hi @carlmontanari ,

There are platforms like Huawei VRP OS that have commands that require confirmation when they are executed.  Here some examples of the confirmation messages:

- r"Warning: The current operation will delete the oldest.*[Y/N]"
- r"Warning: This operation will revert configuration changes to the previous status.*[Y/N]"

Genie/Unicon have a way to manage this.

What have you think about how to manage this in Scrapli? Scrapli have a way to manage this?

Regards

**Comment by carlmontanari** [comment]
https://carlmontanari.github.io/scrapli/user_guide/basic_usage/#handling-prompts
https://github.com/carlmontanari/scrapli/blob/main/examples/read_callback/sync_read_callback.py
